packages/cerebral-provider-firebase/src/remove.js

```diff
@@ -1,13 +1,14 @@
 import {
   createRef
 } from './helpers'
+import {FirebaseProviderError} from './errors'
 
 export default function remove (path) {
   const ref = createRef(path)
-  return new Promise((resolve, reject) => {
-    ref.remove().then(
-      () => resolve({}),
-      (error) => reject({error: error.message})
-    )
-  })
+
+  return ref.remove()
+    .then(() => undefined)
```

Comment:
Not really, `{}` causes a merge to happen, while `undefined` is completely ignored... but `{}` makes you understand the code better so have changed that now :)